src/App.tsx

```diff
@@ -7,13 +7,18 @@ import { useDerivAnalytics, useRedirectToOauth, useTrackjs } from '@/hooks';
 import AppContent from '@/routes/AppContent';
 import { initializeI18n, TranslationProvider } from '@deriv-com/translations';
 import { Loader, useDevice } from '@deriv-com/ui';
+import { URLConstants } from '@deriv-com/utils';
+import useGrowthbookGetFeatureValue from './hooks/custom-hooks/useGrowthbookGetFeatureValue';
 
 const { VITE_CROWDIN_BRANCH_NAME, VITE_PROJECT_NAME, VITE_TRANSLATIONS_CDN_URL } = process.env;
 const i18nInstance = initializeI18n({
     cdnUrl: `${VITE_TRANSLATIONS_CDN_URL}/${VITE_PROJECT_NAME}/${VITE_CROWDIN_BRANCH_NAME}`,
 });
 
 const App = () => {
+    const [ShoudlRedirectToDerivApp, isGBLoaded] = useGrowthbookGetFeatureValue({
```

Comment:
```suggestion
    const [ShouldRedirectToDerivApp, isGBLoaded] = useGrowthbookGetFeatureValue({
```
Typo :-D 
fixing it.